### ISSUE TITLE
Error no such file or directory "firefox"

### ISSUE THREAD

**Issue by compulon** [yeongbin-jo/python-geckodriver-autoinstaller#4]
I get that error. Could it be that Firefox must be installed on the PC where I am running the script?

**Comment by wblondel** [comment]
Yes. Firefox has to be installed if you want to use geckodriver. Same thing for ChromeDriver, you need to have Chrome installed.
Or you can use a Portable version of the browser.

**Comment by nxbx** [comment]
Same issue.

Trying to run from flask app.

Ubuntu 18, Firefox installed.
